src/components/OTP.vue

```diff
@@ -71,11 +72,29 @@
       {{ requestOTPButtonDisplayText }}
     </button>
 
+    <!-- button to resend OTP -->
+    <button
+      @click="sendOTP"
+      class="bg-primary hover:bg-primary-hover text-white uppercase text-lg mx-auto p-4 mt-4 disabled:opacity-50"
+      v-show="isCountdownFinished"
+    >
+      {{ resendOTPButtonText }}
+    </button>
+
     <!-- OTP response message  -->
-    <span v-if="willDisplayOTPMessage" :class="displayOTPMessageClass">{{
-      willDisplayOTPMessage
+    <span v-show="willDisplayOTPMessage" :class="displayOTPMessageClass">{{
+      OTPMessage
     }}</span>
 
+    <!-- timer for resending OTP -->
+    <p
+      v-show="startCountdown"
```

Comment:
This variable should be named something like `isResendCountdownShown` or something.

I think @deepansh96 and @dalmia follow a convention for these boolean variables (I noticed while spying on their PRs). Follow the same convention.